hedera-node/test-clients/src/main/resources/contract/contracts/AssociateContract/AssociateContract.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./HederaTokenService.sol";
+import "./HederaResponseCodes.sol";
+
+contract AssociateContract is HederaTokenService{
+
+    function associateToken(address token) external {
```

Comment:
Please comment or change name to clarify the difference to similar contracts elsewhere

Reply:
Renamed for clarity